boards/arm/stm32/nucleo-l152re/src/stm32_appinitialize.c

```diff
@@ -80,6 +83,52 @@
 int board_app_initialize(uintptr_t arg)
 {
   int ret;
+#ifdef CONFIG_STM32_I2C1
+  FAR struct i2c_master_s *g_i2c1;
```

Comment:
Variables starting with symbol '"g_*" means it is a global variable, I think this is not the case here. Please fix it.

Reply:
fix it.